python/ray/scripts/scripts.py

```diff
@@ -577,6 +577,27 @@ def get_head_ip(cluster_config_file, cluster_name):
     click.echo(get_head_node_ip(cluster_config_file, cluster_name))
 
 
+@cli.command()
+def stack():
+    COMMAND = """
+pyspy=`which py-spy`
+if [ ! -e "$pyspy" ]; then
+    echo "ERROR: Please 'pip install py-spy' first"
+    exit 1
+fi
+export IFS="
```

Comment:
What does this do?

Reply:
It lets you iterate over the lines by line instead of by word.